### PR TITLE
Implement auth and i18n

### PR DESCRIPTION
**Pull request by Plakhotenko**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const client = axios.create({
+  baseURL: 'http://localhost:3001/api',
```
Comment (Rammfall) [inline src/client.ts]: I think needs create .env file and all configurations write there

File: src/components/header/component.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { Container, Typography, Link, Box, List, ListItem } from '@mui/material';
+
+const Header: FC = () => (
```
Comment (Rammfall) [inline src/components/header/component.tsx]: All components directories should have names in Uppercased word. Header

File: src/components/input/component.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { TextField } from '@mui/material';
+import { useField } from 'formik';
+
```
Comment (Rammfall) [inline src/components/input/component.tsx]: Input directory name

File: src/components/registration-form/component.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import React, { FC, useState, useEffect, useRef } from 'react';
```
Comment (Rammfall) [inline src/components/registration-form/component.tsx]: RegistrationForm

**Review by Rammfall — CHANGES_REQUESTED** [review]

File: src/pages/RegistrationForm/component.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+          name: '',
+          email: '',
+          password: '',
+          confirmPassword: '',
+        }}
+        validationSchema={validationSchema}
+      >
+        <Form>
+          <Typography variant="h5" component="h1" mb={2} sx={{ textAlign: 'center' }}>
+            <Translate t="registrationForm.title" />
```
Comment (Rammfall) [inline src/pages/RegistrationForm/component.tsx:69]: Maybe we can change prop name to 'id'?

**Review by Rammfall — CHANGES_REQUESTED** [review]

File: src/pages/RegistrationForm/component.tsx (hunk 5 above)
Comment (Plakhotenko) [reply src/pages/RegistrationForm/component.tsx:69]: It is rather a path, not an id.
Also I'm trying to follow common approach. Like in this library https://react.i18next.com/latest/usetranslation-hook

**Review by Rammfall — APPROVED** [review]

**Pull request merged**